website/homepage/nginx.conf

```diff
@@ -3,44 +3,38 @@ server {
     server_name  localhost;
     absolute_redirect off;
 
-    #access_log  /var/log/nginx/host.access.log  main;
-
     location / {
         root   /usr/share/nginx/html;
         try_files $uri $uri/ $uri/index.html =404;
         index  index.html index.htm;
         autoindex on;
     }
 
-    error_page  404              /404.html;
+    location /introduction_to_blocks {
+        return 301 /quickstart;
+    }
+
+    location /adding_examples_to_your_app {
+        return 301 /key_features;
+    }
+
+    location /embedding_gradio_demos {
+        return 301 /sharing_your_app;
+    }
+
+    location /getting_started {
+        return 301 /quickstart;
+    }
 
-    # redirect server error pages to the static page /50x.html
-    #
+    location /guides.html {
+        return 301 /guides;
+    }
+
+    
+    error_page  404              /404.html;
```

Comment:
Where is this 404.html file? I don't see it. Nor do I see the /50x.html file below.

Navigating manually to `http://localhost:8000/404.html` doesn't work for me either

Reply:
This line isn't doing anything (already existed before), but if we want to create a 404/503 page we can. I can make a simple one in a separate PR to look better than the standard nginx pages.

Reply:
Got it thanks -- agreed we can do in a separate PR, not very important